analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/VerifierTest.cs

```diff
@@ -106,7 +125,7 @@ public class VerifierTest
 
         [TestMethod]
         public void Verify_TwoAnalyzers() =>
```

Comment:
Is this test correct? It says 2 analyzers but we only add one.

Reply:
`WithSnippetCS` scaffolds the first instance as in every other UT and the `.AddAnalyzer(() => new DummyAnalyzerCS()) // Duplicate` adds the duplicate one.

It works as expected, as the `@-1` would not be there otherwise (as seen in every other UT)